packages/esm-patient-tests-app/src/test-results/panel-view/panel-view.component.tsx

```diff
@@ -29,32 +25,39 @@ interface PanelViewHeaderProps {
   totalSearchResults: number;
 }
 
-const PanelView: React.FC<PanelViewProps> = ({ expanded, testUuid, basePath, type, patientUuid }) => {
+const PanelView: React.FC<PanelViewProps> = ({ expanded, patientUuid }) => {
   const { t } = useTranslation();
   const layout = useLayoutType();
   const isTablet = layout === 'tablet';
-  const trendlineView = testUuid && type === 'trendline';
-  const { panels, isLoading, groupedObservations } = usePanelData(patientUuid);
-
+  const { isLoading } = usePanelData(patientUuid);
```

Comment:
This is done in another Test Results component as well, because FilterContext lacks an `isLoading` element. I should just fix that.